app/models/annotation_resource.rb

```diff
@@ -0,0 +1,13 @@
+##
+# Resource used for modeling Annotation content
+class AnnotationResource < Spotlight::Resource
+  self.document_builder_class = AnnotationBuilder
+
+  store :data, accessors: [:annotations]
+
+  def resources
```

Comment:
I'm somewhat at a loss for how we're doing to associate an annotation with an exhibit (since we'd have to hop from annotation -> canvas -> manifest -> exhibit, but all of our associations point the other direction).  I suspect we'll have to do something like #82 and trigger a full annotation reindex whenever a manifest is indexed.

Reply:
Maybe it doesn't need to be that bad; since i /hope/ we can assume there's always a pointer from annotations to manifests (`on/within`), we can do something dumb and not particularly performant to get the exhibit association (see 81be14d).